### ISSUE TITLE
use package.json like docco-husky

### ISSUE THREAD

**Issue by shapeshifta78** [jacobrask/styledocco#27]
Hi there,

I'd like to use docco husky and styledocco together for my projects. husky has a package.json file which contains the project name and I'd like to use that name for styledocco as well and link beetween them via the readme file.
I tried to fetch it with a bash script, but styledocco does not use the name from the variable, it just inserts "true".

Maybe my script is wrong, but when I use echo it outputs sth. like styledocco -name "name from json file" css which is correct.
Here is my script:

```
#!/bin/bash
name=`cat package.json | grep name | sed -e "s/^.*\"\(.*\)\".*$/\"\1\"/"`
echo $name
styledocco -name $name css 
docco-husky js
```

Regards,
Tom

**Comment by shapeshifta78** [comment]
Why does it work if I do: styledocco -n $name css ?

**Comment by jacobrask** [comment]
In the next version of StyleDocco it will look for a name in package.json if you don't specify -name.

**Comment by shapeshifta78** [comment]
\o/ Awesome news! :)
Any plans when it will be available?

**Comment by jacobrask** [comment]
I released 0.4.0 yesterday with this change

**Comment by shapeshifta78** [comment]
Good news! Thank you :)
